toxcore/tox.c

```diff
@@ -72,23 +72,25 @@ typedef struct Messenger Tox;
 
 uint32_t tox_version_major(void)
 {
-    return 0;
+    return TOX_VERSION_MAJOR;
 }
 
 uint32_t tox_version_minor(void)
 {
-    return 0;
+    return TOX_VERSION_MINOR;
 }
 
 uint32_t tox_version_patch(void)
 {
-    return 0;
+    return TOX_VERSION_PATCH;
 }
 
 bool tox_version_is_compatible(uint32_t major, uint32_t minor, uint32_t patch)
 {
-    //TODO
-    return 1;
+    if (major != TOX_VERSION_MAJOR)
+        return 0;
+    else
+        return 1;
 }
```

Comment:
it's like this because if minor is > *_VERSION_MINOR, we should fail. But that's a discussion that hasn't been resolved yet.